client/src/exec.rs

```diff
@@ -162,18 +162,19 @@ async fn exec_and_print(
     Ok(())
 }
 
-fn use_database(sql: &str) -> Option<String> {
+fn parse_use_database(sql: &str) -> Option<String> {
     let sql = sql.trim().trim_end_matches(';');
     if !sql[0..3].to_ascii_lowercase().eq("use") {
         return None;
     }
 
-    let sql = sql[3..].trim();
+    let database = sql[3..].trim();
 
-    if sql.starts_with('"') && sql.ends_with('"') {
-        Some(sql[1..sql.len() - 1].to_string())
+    if database.starts_with('"') && database.ends_with('"') {
+        Some(database.trim_matches('"').to_string())
     } else {
-        Some(sql.to_string())
+        // normalize ident
+        Some(database.to_ascii_lowercase())
```

Comment:
Is it possible to not change the string，because it will be coupled with the server

Reply:
But here the cli needs to prompt the user for the name of the database